src/modules/About/About.tsx

```diff
@@ -2,13 +2,128 @@ import React from "react";
+              xmlns="http://www.w3.org/2000/svg"
+              className="lg:mb-24"
+            >
+              <g id="ChatsCircle">
+                <path
+                  id="Vector"
+                  d="M72.4348 58.5397C74.1979 54.9557 75.0767 51.0013 74.9974 47.0079C74.9181 43.0144 73.8831 39.0981 71.979 35.5869C70.075 32.0756 67.3574 29.0717 64.0537 26.8267C60.7501 24.5817 56.9566 23.161 52.991 22.6835C51.6759 19.6176 49.7604 16.846 47.3574 14.5319C44.9545 12.2178 42.1127 10.408 38.9995 9.20921C35.8863 8.01041 32.5646 7.44682 29.2302 7.55165C25.8958 7.65648 22.6161 8.42761 19.5843 9.8196C16.5526 11.2116 13.8301 13.1963 11.5772 15.6568C9.32441 18.1172 7.5868 21.0037 6.4668 24.1461C5.3468 27.2885 4.86706 30.6233 5.05584 33.954C5.24463 37.2847 6.09812 40.544 7.56604 43.5397L5.17229 51.921C4.95868 52.6708 4.94933 53.4641 5.14518 54.2187C5.34104 54.9733 5.735 55.6619 6.28629 56.2132C6.83758 56.7645 7.52618 57.1585 8.28082 57.3543C9.03546 57.5502 9.82873 57.5408 10.5785 57.3272L18.9598 54.9335C21.4889 56.1752 24.2094 56.9813 27.0067 57.3178C28.3401 60.4419 30.2962 63.2614 32.7555 65.6044C35.2148 67.9474 38.1257 69.7646 41.3107 70.9452C44.4957 72.1258 47.8878 72.6449 51.28 72.471C54.6723 72.297 57.9935 71.4336 61.041 69.9335L69.4223 72.3272C70.1721 72.5408 70.9654 72.5502 71.72 72.3543C72.4747 72.1585 73.1632 71.7645 73.7145 71.2132C74.2658 70.6619 74.6598 69.9733 74.8556 69.2187C75.0515 68.4641 75.0421 67.6708 74.8285 66.921L72.4348 58.5397ZM67.3535 58.9585L69.7129 67.2116L61.4598 64.8522C60.831 64.6754 60.158 64.7517 59.5848 65.0647C54.9763 67.5719 49.5678 68.1698 44.523 66.7298C39.4782 65.2898 35.2003 61.9269 32.6098 57.3647C36.0331 57.0082 39.3457 55.9481 42.34 54.2508C45.3342 52.5535 47.9453 50.2558 50.0096 47.5017C52.0738 44.7476 53.5466 41.5967 54.3355 38.2465C55.1245 34.8963 55.2125 31.4193 54.5942 28.0335C57.5899 28.7395 60.3822 30.1273 62.7538 32.089C65.1254 34.0506 67.0122 36.5332 68.2675 39.3433C69.5227 42.1535 70.1124 45.2154 69.9908 48.2908C69.8691 51.3661 69.0394 54.3719 67.566 57.0741C67.2505 57.6499 67.1742 58.3268 67.3535 58.9585Z"
+                  fill="white"
+                />
+              </g>
+            </svg>
+
+            <div className="inline-flex max-w-fit flex-col items-start justify-start gap-4">
+              <div className="text-5xl font-normal text-white">
+                <h2 className="lg:text-[48px] text-[30px]">
+                  Exchange Stories. Create New Vision.
+                </h2>
+              </div> 
+              <div className="text-xl text-left font-normal text-rose-100">
+                <p className="max-w-screen text-[16px] lg:w-[873px] lg:h-[72px] lg:text-[20px]">
+                  BarCamp is where people gather to share their ideologies, and
+                  with those ideas; new visions and innovations will come to
+                  blossom into new projects.
+                  <br />
+                </p>
+              </div>
+            </div>
+          </div>
```

Comment:
![image](https://github.com/barcamp-bangkhen-11/bcbk11-frontend/assets/70707895/338775e9-12ad-4662-af63-e7e1ca6badb5)

text overflow out of screen in screen size width about 1024px.
maybe not fix the container size try to add padding instead